### PR TITLE
Add sigma prime security testing CI

### PR DESCRIPTION
**Pull request by strokovok**

### CODE REVIEW

**Comment by sept-en** [comment]
Is it still a draft or ready is it ready for a review?

**Comment by SouadAli10** [comment]
this pr is ready

File: .github/workflows/sigma-prime.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            -f sigma-prime/tests/Dockerfile \
+            -t "$TESTING_IMAGE_NAME" \
+            ./sigma-prime
+
+      - name: Run tests
+        run: |
+          docker run \
+            --rm \
+            -v "$(pwd)/.:/repo/rainbow-bridge" \
+            -e SOME_VARIABLE=SOME_VALUE \
```
Comment (sept-en) [inline .github/workflows/sigma-prime.yml:41]: What's it? I believe it was left here by mistake.

```suggestion
```

**Review by sept-en — CHANGES_REQUESTED** [review]

**Review by karim-en — APPROVED** [review]
LGTM, just one comment which already mentioned by @sept-en

File: .github/workflows/sigma-prime.yml (hunk 1 above)
Comment (strokovok) [reply .github/workflows/sigma-prime.yml:41]: Well I left that as a placeholder for @SouadAli10 to substitute the parameters.
Idk which particular parameters are needed)

**Review by sept-en — APPROVED** [review]
The wireframe for these tests look okayish, though it's not fully working as expected. Follow modifications will be added to the testing-repo first and then some changes will be reflected here too. As for now, merging and then the suite will be updated after changes in testing-repo.

**Comment by SouadAli10** [comment]
> The wireframe for these tests look okayish, though it's not fully working as expected. Follow modifications will be added to the testing-repo first and then some changes will be reflected here too. As for now, merging and then the suite will be updated after changes in testing-repo.
sure these tests that were provided by sigma prime I did not modify anything in them

**Pull request merged**